wordcount_and_median.py

```diff
@@ -43,46 +42,32 @@ def __cleaner_lister(incoming_string):
     outgoing_list = incoming_string.split()
     return outgoing_list
 
-     
 
 def count_writer(output_folder, wc_out_file, fulldict_cw):
-    with open(output_folder+'/'+wc_out_file, 'w') as d:
-        for item in sorted( fulldict_cw.keys() ):
-            out_line = str(item)+'\t'+str( fulldict_cw[item] )+'\n'
-            d.write(out_line)
-        d.closed
+    with open(os.path.join(output_folder, wc_out_file), 'w') as d:
+        writer = csv.writer(d, delimiter='\t')
+        for word, count in sorted(fulldict_cw.items()):
+            writer.writerow([word, str(count)])
```

Comment:
Two minor refactors here:

1) Since we're basically emitting a ``tsv`` file, we can use the ``csv`` module.  Not really much of a savings in this case in terms of code simplicity, but it's a generally useful module.

2) When iterating over the keys and values in a dictionary, it's usually easiest to say something like ``for key, val in mydict.items()``, so I changed this to use that idiom.